copier.yaml

```diff
@@ -11,7 +11,8 @@ _envops:
   variable_start_string: "{{"
 
 _tasks:
-  - '(cd apps/{{api}}; npx npm-add-script -f -k bootstrap -v "talos put --api \${{api|upper}}_BASE --endpoint \$SPARQL_ENDPOINT --user \$SPARQL_USER --password \$SPARQL_PASSWORD")'
+  - cd apps/{{api}} || cd apps\{{api}}
+  - '(npx npm-add-script -f -k bootstrap -v "talos put --api \${{api|upper}}_BASE --endpoint \$SPARQL_ENDPOINT --user \$SPARQL_USER --password \$SPARQL_PASSWORD")'
```

Comment:
Will this work when separated? I would expect each task to be executed in the root. 

Thus, you `cd` and then won't the next line executes again in the original dir?